src/scss/06_components/paragraphs/_react-search.scss

```diff
@@ -102,3 +102,72 @@ fieldset .hdbt-search--react__checkbox:not(:first-of-type) {
     }
   }
 }
+
+.component--hdbt-search--react .component__container {
+  background-color: $color-silver-light;
+}
+
+.components--upper .component--hdbt-search--react .component__container {
+  margin-left: $spacing;
+  margin-right: $spacing;  
+  padding: $spacing-and-half;
+
+  @include breakpoint($breakpoint-m) {    
+    margin-left: $spacing-double;
+    margin-right: $spacing-double;
+    padding: $spacing-double;  
+  }
+
+  @include breakpoint($breakpoint-l) {
+    margin: 0;
+  }
+}
+
+.has-sidebar .components--upper .component--hdbt-search--react {
+  @include breakpoint(max-width $breakpoint-l) {
+    @include component-negative-side-margin;
+  }
+}
+
+.hdbt-search--react__form-container {
+  border-bottom: none;
+}
+
+.hdbt-search--react__form-title {
```

Comment:
This class should be Ploughing search specific.

Reply:
Changed.